lnode-src/duckcoin.go

```diff
@@ -382,7 +382,7 @@ func reCalcDifficulty() {
 	}
 	avg /= i
 	avgDur := time.Duration(avg)
-	fmt.Println("The average duration between blocks for the past 100 blocks was:", avgDur)
+	fmt.Println(fmt.Sprintf("The average duration between blocks for the past %d blocks was: %f", ReCalcInterval, avgDur))
```

Comment:
Has this been tested?

Reply:
oops

Reply:
(I'm just not certain about the %f flag)

Reply:
yeah it's wrong!